backends-velox/src/main/scala/org/apache/spark/sql/execution/SparkWriteFilesCommitProtocol.scala

```diff
@@ -97,8 +97,11 @@ class SparkWriteFilesCommitProtocol(
     }
   }
 
-  def abortTask(): Unit = {
+  def abortTask(writePath: String): Unit = {
     committer.abortTask(taskAttemptContext)
+
+    val tmpPath = new Path(writePath)
+    tmpPath.getFileSystem(taskAttemptContext.getConfiguration).delete(tmpPath, false)
```

Comment:
`delete(tmpPath, true)` ? the boolean param means if delete files recursive when the path is a directory.  IIUC, `tmpPath` is a directory.

Reply:
@ulysses-you Good catch. Updated.